### PR TITLE
improv: replace arg in LOG_FUNCTION_* to __func__

### PR DESCRIPTION
**Pull request by Akylzhan**
fixes https://github.com/Koihik/LuaFormatter/issues/91

### CODE REVIEW

File: src/FormatVisitor.cpp [hunk 1]
```diff
@@ -18,19 +18,19 @@ using namespace antlr4;
...
-#define LOG_FUNCTION_END(arg)                            \
+    cout << __func__ << " begin.." << endl
+#define LOG_FUNCTION_END()                               \
     for (int i = 0; i < logIndentSize; i++) cout << " "; \
     logIndentSize--;                                     \
-    cout << arg << " end.." << endl
+    cout << __func__ << " end.." << endl
 #else
 #define LOG(arg)
 #define LOGVAR(arg)
```
Comment (tammela) [inline src/FormatVisitor.cpp]: remove `arg` from those as well

Comment (Akylzhan) [reply src/FormatVisitor.cpp]: also, should I remove parentheses?

Comment (tammela) [reply src/FormatVisitor.cpp]: not needed

**Comment by tammela** [comment]
Please squash all commits

**Comment by Akylzhan** [comment]
@tammela done

**Pull request merged**